### DISCUSSION TITLE
Pro.Extensions js error not found

### DISCUSSION THREAD

**Discussion by belalAhmed** [serenity-is/Serenity#6015] [Help]
is any one know why this error happened

Pro.Extensions/index.js?v=2021100704 not found 

![image](https://user-images.githubusercontent.com/3522191/136384245-d5098093-b885-4e7c-b317-e4591cdf9c20.png)

**Comment by VictorTomaili** [comment]
It's our premium package, if you don't have a licence to use this library, you cannot use it. [Please check this](https://serenity.is/docs/startsharp/README). If you have a licence, please open an issue on the StartSharp repository.

**Reply by belalAhmed** [reply]
how i can remove it from the project !!

**Reply by go2ismail** [reply]
Hi,

You can remove it:

![image](https://user-images.githubusercontent.com/2275051/149112244-aeac64eb-71bf-47a1-9109-d73a99f2b18d.png)
